teamsupport/models.py

```diff
@@ -43,7 +49,9 @@ def __init__(self, ticket_id=None, data=None):
             raise MissingArgumentError(
                 "__init__() needs either a 'ticket_id' or 'data' argument "
                 '(neither given)')
-        self.id = self.TicketID
+
+        id_attr = 'ID' if 'ID' in self.data else 'TicketID'
```

Comment:
When is this `ID` instead of `TicketID`?

Reply:
Sometimes. As I understand, when Create result is returned, it is 'ID'. When we search results, this is `<entity_name>ID`. Weird, yes.